(next-urql) - Never suspend option (#923)
FormidableLabs/urql: docs/advanced/server-side-rendering.md

Unless the component that is being wrapped already has a `getInitialProps` method, `next-urql` won't add its own SSR logic, which automatically fetches queries during
server-side rendering. This can be explicitly enabled by passing the `{ ssr: true }` option as a second argument to `withUrqlClient`.

> When you are using `getStaticProps`, `getServerProps`, ... You should opt-out of `Suspense` by setting the `neverSuspend` option to `true` in
> your `withUrqlClient`.

Reviewer: ```suggestion
When you are using `getStaticProps`, `getServerSideProps`, or `getStaticPaths`,  you should opt-out of `Suspense` by setting the `neverSuspend` option to `true` in your `withUrqlClient` configuration.
your `withUrqlClient`.
```

> This because when we are doing a prepass of your component tree we can't know what these functions will inject in the props and this injection

Reviewer: ```suggestion
During the prepass of your component tree `next-urql` can't know what these functions will inject in the props of your page. This injection
```

> could change the `variables` used in your `useQuery`, ... Making these throw during the subsequent `toString` pass which isn't supported in React 16.

Reviewer: ```suggestion
could change the `variables` used in your `useQuery`. This will lead to error being thrown during the subsequent `toString` pass, which isn't supported in React 16.
```
